utils/process-stats.py

```diff
@@ -60,17 +133,36 @@ def resource_stats_generate(filename: str, data: Dict) -> None:
 
 
 if __name__ == '__main__':
-    if len(sys.argv) < 3:
-        print(f'Usage: {sys.argv[0]} output-file input-file [input-file ...]')
-        sys.exit(1)
+    parser = argparse.ArgumentParser(
+        description='Generate resource usage charts.'
+    )
+    parser.add_argument(
+        '--aggregate', action='store_true', help='generate aggregate charts'
+    )
+    parser.add_argument(
+        '-o', '--output', required=True, help='file to write an HTML result'
+    )
```

Comment:
Since this argument is required, it seems odd to have the `-o` prefix for the option. Perhaps instead this could just be
```
    parser.add_argument(
        'output', required=True, help='file to write an HTML result'
    )
```

This would also allow for existing callers of `process_stats.py` to not have to be updated.

Reply:
I find it very difficult to have more than one positional argument with argparse.  I tried and gave up adding a required optional argument instead.

Reply:
Sorry for the bad suggestion, it should have been 
```
    parser.add_argument(
        'output', help='file to write an HTML result'
    )
```

Reply:
This one works.  Though I'm still not a huge fan of multiple positional arguments.
@dceara do you have a preference?

Reply:
I prefer the explicit `-o` but I'm OK either way.

Reply:
We discussed a bit offline, we'll go for `-o`.